client/src/Controller/SettingsController.php

```diff
@@ -110,19 +110,20 @@ public function passwordEditAction(Request $request)
 
         $form = $this->createForm(FormDir\ChangePasswordType::class, $user, [
             'mapped' => true,
-            'error_bubbling' => true
+            'error_bubbling' => true,
         ]);
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
             $plainPassword = $request->request->get('change_password')['password']['first'];
-            $this->restClient->put('user/' . $user->getId() . '/set-password', json_encode([
+            $this->restClient->put('user/'.$user->getId().'/set-password', json_encode([
                 'password_plain' => $plainPassword,
             ]));
             $request->getSession()->set('login-context', 'password-update');
 
             $successRoute = $this->getUser()->isDeputyOrg() ? 'org_settings' : 'account_settings';
-            return $this->redirect($this->generateUrl($successRoute));
+
+            return $this->redirect('/logout');
```

Comment:
Presumably this changes the user flow for the redirect back? So may want to check that with product side.

Can you avoid needing to do this by pushing the isDeputyOrg and generate url up before the password reset or is there some state required? 

Also, presumably the successRoute is no longer used so doesn't need to be set at all.

Reply:
As far as I know this doesn't actually change the user flow. Whenever a password was edited it would always log them out as that's just a security setting of Symfony. The reason we had to back the change sot he redirect was to the `/logout` page was because removing the `password hash` from the JSON response made it so once a password was edited it would no longer log them out.

So this redirect change just keeps that user flow the same :)